### DISCUSSION TITLE
Install app link not available

### DISCUSSION THREAD

**Discussion by SootR** [photoprism/photoprism#2988] [Q&A]
Hello,

I have installed Photoprism following the tutorial for Docker Compose (https://docs.photoprism.app/getting-started/docker-compose/) with only modifying the user credentials in the default configuration.

So far everything is working fine, but on my Android mobile (Pixel 6) I can't find the "Install app" link when browsing via Chrome.

Is there anything to enable to access the PWA?

Thanks in advance!

Soot

**Comment by lastzero** [comment]
It requires https with a valid certificate. But you can also "just" add it to your home screen.

**Reply by SootR** [reply]
Understood. I was still testing locally. Then I'll setup Apache and create the certificates.

Thanks for the hint!

**Comment by rickysarraf** [comment]
If you get it to work with self signed certificates, please do share here

s3nt fr0m a $martph0ne, excuse typ0s

**Reply by SootR** [reply]
Photoprism is working with Letsencrypt certificates just fine.

But still I can't see the "Install app" link in the menu. Any ideas?
